src/OmniSharp/Api/TestContext/AspNet5TestCommandProvider.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace OmniSharp
+{
+    public class AspNet5TestCommandProvider : ITestCommandProvider
+    {
+        public string GetTestCommand(TestContext testContext)
+        {
+            if (new FileInfo(testContext.ProjectFile).Extension != ".json")
+            {
+                return null;
+            }
+
+            // Find the test command, if any and use that
+            var node = testContext.SyntaxNode;
```

Comment:
Why not figure this out outside of the ASP.NET specific implementation? and pass in the ISymbol instead of the SyntaxNode.